cmd/buildah/containers_test.go

```diff
@@ -43,3 +44,64 @@ func TestTemplateOutputInvalidFormat(t *testing.T) {
 		t.Fatalf("expected error invalid format")
 	}
 }
+
+func TestContainerTemplateOutputInexistenceField(t *testing.T) {
```

Comment:
I was not familiar with the work Inexistence, I would have used Nonexistent, but since it is a real word, I am fine with it.